pulseapi/entries/serializers.py

```diff
@@ -67,6 +69,7 @@ class EntrySerializer(serializers.ModelSerializer):
         required=False
     )
 
+    # QUEUED FOR DEPRECATION: Use the `related_creators` property instead.
```

Comment:
can you add the issue that we filed for this? (if there is none, can you file it? =)